app/models/foreman_docker/docker.rb

```diff
@@ -131,17 +132,19 @@ def bootstrap(args)
     end
 
     def client
-      @client ||= ::Fog::Compute.new(
+      cli = ::Fog::Compute.new(
```

Comment:
This would make a new connection on every single request, why not using the instance variable as any other fog connection in Foreman?